erpnext/stock/doctype/delivery_trip/delivery_trip.py

```diff
@@ -204,6 +200,21 @@ def rearrange_stops(self, optimized_order, start):
 
 		self.delivery_stops[start:start + len(stops_order)] = stops_order
 
+	def get_client(self):
```

Comment:
This is a very generic function, and should not be restricted to the Delivery Trip class. Otherwise, you'd need to define this for every doctype where it's needed.

Reply:
Direction API Integration is just used in Delivery Trip, so making it a class function